src/ornament.ts

```diff
@@ -177,54 +142,50 @@ export class Ornament extends Modifier {
   constructor(type: string) {
     super();
 
+    // Default position ABOVE
+    this.position = ModifierPosition.ABOVE;
+    if (Ornament.ornamentRight.indexOf(type) >= 0) {
+      this.position = ModifierPosition.RIGHT;
+    }
+    if (Ornament.ornamentLeft.indexOf(type) >= 0) {
+      this.position = ModifierPosition.LEFT;
+    }
     this.type = type;
     this.delayed = false;
 
     this.renderOptions = {
-      fontScale: Tables.NOTATION_FONT_SCALE,
       accidentalLowerPadding: 3,
       accidentalUpperPadding: 3,
     };
 
-    this.ornament = Tables.ornamentCodes(this.type);
-
-    // new ornaments have their origin at the origin, and have more specific
-    // metrics.  Legacy ornaments do some
-    // x scaling, and have hard-coded metrics
-    const metrics = this.getMetrics();
-
     // some jazz ornaments are above or below depending on stem direction.
     this.adjustForStemDirection = false;
 
-    // some jazz ornaments like falls are supposed to overlap with future bars
-    // and so we report a different width than they actually take up.
```

Comment:
Interesting comment that is now deleted. :-) I am not an expert with jazz ornaments, so we'll have to get these reviewed by experts later on. How will we handle this overlap?